### ISSUE TITLE
Need to re-pair Philips Hue Bridge

### ISSUE THREAD

**Issue by mthoretton** [bwssytems/ha-bridge#669]
Hi!

I'm running HA Bridge on a raspi3 as a service/deamon. I've linked Harmony, Domoticz and Philips Hue hub. Everything is working quite well.
But, when I reload the habridge service (or I reboot the raspi) I have to re-pair the Philips Hue bridge by pushing the button. The rest of the configuration (Domoticz and Harmony) still work.

I didn't find any related issue in the closed issues neither in the doc and it doesn't look like the normal behavior. (I'm very sorry if I missed it).

Do you have any idea? This is I guess the most relevant line in the logs:
`Retry Register to Hue for Hue (physical) returned error: [{"error":{"type":1,"address":"/","description":"unauthorized user"}}]`

btw thanks a lot for your work, habridge is the "centrepiece" of my domotic setup =D

**Comment by bwssytems** [comment]
What version are you running?

**Comment by mthoretton** [comment]
`ha-bridge-4.5.1.jar`, I missed the last one (`4.5.5`) but according to the changelog nothing that could help with my issue. (I'll get the last version when I come back home and try it out just in case). I could also run habridge in debug mode to see if I can find more information.

**Comment by bwssytems** [comment]
Well, thought I had that fixed in 4.5.1. Will look into it.

**Comment by bwssytems** [comment]
@mthoretton Could you give me all the log messages? I need to see what is before that error message for retry.

Thanks!

**Comment by bwssytems** [comment]
So, did not implement saving. Took a a look how to do it and it was pretty simple.

**Comment by mthoretton** [comment]
Hey, sorry for the delay. I tried a few things and now: it's working with `4.5.5` but only with `-Ddev.mode=true` argument :p Otherwise those are the errors I get in the UI (nothing in the logs).
<img width="573" alt="screen shot 2017-06-19 at 22 48 26" src="https://user-images.githubusercontent.com/6221533/27305584-7121b352-5542-11e7-8c09-2ff2b0995ef8.png">

Is this commit supposed to fix the issue? https://github.com/bwssytems/ha-bridge/commit/220e337d0884f13bed81660b8a3733189d3e6f72

Thanks

**Comment by bwssytems** [comment]
Dev mode true is a bad option as that does not connect to a real hue just internal data. Wait until the next release

**Comment by Khengsoon** [comment]
I noticed something, this happen when my Pi is on specific IP only. previously I did something like this, I unplugged the SD card and run on another Pi, existing Pi I didn't power off, and my Pi is set to fixed IP so I believe 2 box is having same IP. starting from then my google home display 3 devices every time I do pairing, 

Now I change my box IP to a new IP and shutdown another box, the issue is gone. This only happened to Google Home, but not Amazon Echo.

Regards,
Kheng Soon
